warble/warble_cli.h

```diff
@@ -11,6 +11,7 @@ DEFINE_string(warble, "", "Creates a new warble with the given text");
 DEFINE_string(reply, "", "Indicates that the new warble is a reply to the given id");
 DEFINE_string(follow, "", "Starts following the given username");
 DEFINE_string(read, "", "Reads the warble thread starting at the given id");
+DEFINE_string(store, "", "Stores to the given file name");
```

Comment:
the store flag should be given to the kvstore server at the command line, not the warble client.

Reply:
Oh, that makes so much more sense now. On it